spec/marten/cli/manage/command/collect_assets_spec.cr

```diff
@@ -1,4 +1,5 @@
 require "./spec_helper"
+require "digest/md5"
```

Comment:
This requirement is not needed technically since Marten already requires `digest/md5`!